rosdep/python.yaml

```diff
@@ -1057,6 +1057,11 @@ python-oauth2:
   debian: [python-oauth2]
   fedora: [python-oauth2]
   ubuntu: [python-oauth2]
+python-oauth2client:
+  debian: [python-oauth2client]
+  fedora: [python-oauth2client]
+  ubuntu:
+    trusty: [python-oauth2client]
```

Comment:
This package might actually start at 14.10. I just tried to install it inside a 13.10 docker and the package couldn't be found (even though python-oauth2 could be).